common/kvstore/src/main/java/org/apache/spark/util/kvstore/KVStoreIterator.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  */
 @Private
-public interface KVStoreIterator<T> extends Iterator<T>, AutoCloseable {
+public interface KVStoreIterator<T> extends Iterator<T>, Closeable {
```

Comment:
Weird one, but it warns on extending `AutoCloseable` without explicitly narrowing the exceptions that `close()` throws to not include `InterruptedException`.